common/seeder/seed-data.js

```diff
@@ -574,6 +586,17 @@ const posts = [
   }
 ];
 
+const comments = [
+  {
+    type: 'createComment',
+    creator: users.luke,
```

Comment:
Should this be randomized?

Reply:
I prefer static test data to always know which users will have no/few/many posts/comments/communities.

We could add a set of randomized comments involving only a subset of users. But for now all I'm looking for is a single comment.